### PR TITLE
[OC-1201] Inform the template when lttd has expired

### PR DESCRIPTION
**Pull request by quinarygio**
Release notes :

In Tasks section :
[OC-1201] Inform the template when lttd has expired

[OC-1201]: https://opfab.atlassian.net/browse/OC-1201

### CODE REVIEW

**Review by freddidierRTE — CHANGES_REQUESTED** [review]
I change the specification :  Change the method setResponseEnabled(boolean) to setLttdExpired()  . It is called only when lttd expired .

Add documentation for this new method in src/docs/asciidoc/reference_doc/response_cards.adoc

File: src/test/utils/karate/businessconfig/resources/bundle_defaultProcess/template/fr/question.handlebars [hunk 1]
```diff
@@ -82,6 +83,19 @@
 
   }
 
+  templateGateway.setResponseEnabled = function(enabled) {
+      const responseEnabled = document.getElementById('responseDisabled');
+      if (responseEnabled != null) {
+        if (!enabled) {
+            responseEnabled.innerHTML="Le temps de réponse est expiré";
+            templateGateway.lockAnswer();
```
[the remark this answers is not in the document]
Comment (quinarygio) [reply src/test/utils/karate/businessconfig/resources/bundle_defaultProcess/template/fr/question.handlebars]: Done

File: src/test/utils/karate/businessconfig/resources/bundle_defaultProcess/template/fr/question.handlebars [hunk 2]
```diff
@@ -82,6 +83,19 @@
...
+        } else {
+            responseEnabled.innerHTML="";
+            templateGateway.unlockAnswer();
```
[the remark this answers is not in the document]
Comment (quinarygio) [reply src/test/utils/karate/businessconfig/resources/bundle_defaultProcess/template/fr/question.handlebars]: Done

**Comment by quinarygio** [comment]
> I change the specification : Change the method setResponseEnabled(boolean) to setLttdExpired() . It is called only when lttd expired .
> 
> Add documentation for this new method in src/docs/asciidoc/reference_doc/response_cards.adoc

Done

**Pull request merged**